app/controllers/application_controller.rb

```diff
@@ -49,8 +49,9 @@ def set_surrogate_key(*surrogate_keys)
     response.headers['Surrogate-Key'] = surrogate_keys.join(' ')
   end
 
-  def redirect_to_root
-    redirect_to root_path
+  def redirect_to_signin
+    response.headers["Cache-Control"] = "private, max-age=60"
```

Comment:
Shouldn’t the age be 0?

Reply:
Updated to 0.
I was hoping to allow private caching (not on fastly but in the browser, proxy etc) of unauthenticated requests. I wouldn't have much impact either way.